$OpenBSD: patch-source_genie_c,v 1.1 2021/11/07 20:33:01 bcallah Exp $

Ignore usleep(3) return value.
It can only fail with EINTR and shouldn't stop execution.

Index: source/genie.c
--- source/genie.c.orig
+++ source/genie.c
@@ -7204,7 +7204,7 @@ genie_down_sema (NODE_T * p)
         }
         UNLOCK_THREAD;
 /* Waiting a bit relaxes overhead */
-        ASSERT (usleep (10) == 0);
+        usleep (10);
         LOCK_THREAD;
 /* Garbage may be collected, so recalculate 'k' */
         k = DEREF (A68_INT, &s);
